### DISCUSSION TITLE
Error for testPriceFeedVersionIsAccurate

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2641] [Q&A]
![Screenshot (52)](https://github.com/user-attachments/assets/f383553c-95a5-423a-8a8d-b146547c961b)

**Comment by Shiva-Sai-ssb** [comment]
@prakharmishra3956 
You are running the wrong command.
The correct command is 
```
forge test --mt testPriceFeedVersionIsAccurate
```

**Reply by prakharmishra3956** [reply]
No, that also did not run the correct one was **--match-test**, it was compiled later on using this.

**Reply by Shiva-Sai-ssb** [reply]
Hey @prakharmishra3956, `--mt` is just a shorthand for `--match-test`, so they should work the same. 
Not sure why it didn’t run for you.

**Comment by EngrPips** [comment]
Interesting.
